blocks/share-bar-block/package.json

```diff
@@ -19,13 +20,8 @@
     "url": "ssh://git@github.com/WPMedia/arc-themes-blocks.git",
     "directory": "blocks/share-bar-block"
   },
-  "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1",
-    "lint": "eslint --ext js --ext jsx features"
-  },
   "peerDependencies": {
-    "@wpmedia/engine-theme-sdk": "*",
-    "@wpmedia/news-theme-css": "*"
+    "@wpmedia/arc-themes-components": "*"
```

Comment:
small note: We should also have arc proptypes as a peer dependency here. Good work documenting what you have so far with arc themes components